src/is_border_vertex.cpp

```diff
@@ -40,7 +40,7 @@ npe_arg(f, dense_int, dense_long, dense_longlong)
 
 npe_begin_code()
   assert_valid_23d_tri_mesh(v, f);
-  const auto res = igl::is_border_vertex(v, f);
+  const auto res = igl::is_border_vertex(f);
```

Comment:
Change a result of this PR: https://github.com/libigl/libigl/pull/1610

Function taking V and F is deprecated. We don't change our (your 😄  ) API to ensure no breaking change. V is simply no longer used.

Reply:
yup